tests/test_telemetry.py

```diff
@@ -597,6 +598,104 @@ def test_app_started_product_info(self):
                     "appsec" in products
                 ), "Product information is not accurately reported by telemetry on app-started event"
 
+    @bug(library="java", reason="APMAPI-969")
+    def test_config_telemetry_completeness(self):
+        """
+        Assert that config telemetry is handled properly by telemetry intake
+
+        ⚠️ Did this test just fail? Read here! ⚠️
+        Some files are manually copied from dd-go from/to the following paths using tests/telemetry_intake/update.sh
+        from: https://github.com/DataDog/dd-go/blob/prod/trace/apps/tracer-telemetry-intake/telemetry-payload/static/
+        to: tests/telemetry_intake/static
+
+        If this test fails, it means that a telemetry key was found in config telemetry that does not
+        exist in any of the files listed above in dd-go
+        The impact is that telemetry will not be reported to the Datadog backend won't be unusable
+
+        To fix this, you must update dd-go to either
+        1) Add an exact config key to match config_norm_rules.json
+        2) Add a prefix that matches the config keys to config_prefix_block_list.json
+        3) Add a prefix rule that fits an existing prefix to config_aggregation_list.json
+        4) (Discouraged) Add a language-specific rule to <lang>_config_rules.json
+
+        Once dd-go is updated, you can copy over the files to this repo and merge them in as part of your changes
+        This can be done by running the following from the src root
+
+        Usage: ./tests/telemetry_intake/update.sh
+        """
+
+        def lowercase_obj(obj):
+            if isinstance(obj, dict):
+                return {k.lower() if isinstance(k, str) else k: lowercase_obj(v) for k, v in obj.items()}
+            elif isinstance(obj, list):
+                return [lowercase_obj(item) for item in obj]
+            elif isinstance(obj, str):
+                return obj.lower()
+            else:
+                return obj
+
+        def load_telemetry_json(filename):
+            with open(f"tests/telemetry_intake/static/{filename}.json", encoding="utf-8") as fh:
+                return lowercase_obj(json.load(fh))
+
+        def get_all_keys_and_values(*objs):
+            result = []
+            for obj in objs:
+                if obj is not None:
+                    if isinstance(obj, dict):
+                        result.extend(list(obj.keys()))
+                        result.extend(list(obj.values()))
+                    elif isinstance(obj, list):
+                        result.extend(obj)
+                    else:
+                        logger.error(f"Unexpected type in concat: {type(obj).__name__}")
```

Comment:
If this occurs, I expect the test to fail due to missing config keys, so not throwing here